docker/puppetserver/docker-entrypoint.d/80-ca.sh

```diff
@@ -60,11 +62,15 @@ else
       fi
 
       timestamp="$(date '+%Y-%m-%d %H:%M:%S %z')"
-      ca_name="Puppet Enterprise CA generated on ${HOSTNAME} at $timestamp"
+      ca_name="Puppet CA generated on ${HOSTNAME} at $timestamp"
 
       # See puppet.conf file for relevant settings
       puppetserver ca setup \
           --ca-name "$ca_name" \
           --config /etc/puppetlabs/puppet/puppet.conf
+
+  elif [ $new_ca_exists -eq 1 ] && [ $old_ca_exists -eq 0 ]; then
+      # Legacy CA upgrade
+      puppetserver ca migrate
```

Comment:
I know we recently talked about how you don't need the `--config` flag to run this, but I noticed that we do pass the `--config` flag for the `setup` command. Is the `setup` command just being super explicit, or are we doing some custom thing that means we should pass it here too?

Reply:
The `setup` command is, afaict, being super explicit. However, that branch also writes to that config file so it seemed fine to leave it explicit. We don't touch the config as part of the migration and it seemed fine to leave it implicit there. But I'm happy either removing the setup's `--config` flag usage or adding it here.